### PR TITLE
Config eslint

### PR DESCRIPTION
**Pull request by Vawnndv**
Config ESlint

### CODE REVIEW

**Comment by LTPhiHung** [comment]
don't come in

**Comment by Vawnndv** [comment]
@ChauHoangTan Please help me review this pull request.
To confirm with Mr. @LTPhiHung

**Review by ChauHoangTan — COMMENTED** [review]
first review

File: frontend/package.json
Comment (Vawnndv) [inline frontend/package.json:1]: Hi @ChauHoangTan  help me review this file

**Review by Vawnndv — COMMENTED** [review]
It's ok to merge

**Pull request merged**